src/recording-playback.md

```diff
@@ -0,0 +1,247 @@
+Viewing session recordings in-browser
+=====================================
+
+Guacamole supports [recording activity within remote desktop sessions](graphical-recording)
+such that it can be played back and reviewed later. Graphical recordings can be
+converted to video [using the `guacenc` tool](graphical-recording) (part of
+[guacamole-server](building-guacamole-server)) or can be played back directly
+in the browser in their native format using Guacamole itself. This has several
+benefits:
+
+* Recordings can be played back while the session is underway.
+* Recordings need not be re-encoded as traditional video, an
+  intensive process that often results in a larger file.
+* It is very easy to locate and playback the recording for a session when doing
```

Comment:
Minor nitpick: I think this should be "play back" when it's a verb. "playback" is only a noun as far as I can tell.

Reply:
Ah, yep - good catch.

Reply:
Fixed via rebase.